pkg/app/server.go

```diff
@@ -31,6 +31,8 @@ import (
 	"time"
 
 	"gopkg.in/yaml.v3"
+	discovery2 "k8s.io/client-go/discovery"
```

Comment:
Can we rename this to something more meaningful?
```suggestion
	clientdiscovery "k8s.io/client-go/discovery"
```

or similar?

Reply:
Ah, missed it.